Applications/Games/League of Legends/Client/script.json

```diff
@@ -1,5 +1,5 @@
 {
-	"scriptName": "BETA Client",
+	"scriptName": "Client",
 	"compatibleOperatingSystems": ["MACOSX", "LINUX"],
 	"testingOperatingSystems": ["MACOSX", "LINUX"],
```

Comment:
If the script works you should set `testingOperatingSystems` to `[]`, otherwise the script will only be found in the applications list if  `testing` was selected.